multicluster/build/yamls/antrea-multicluster-member.yml

```diff
@@ -1,872 +1,3 @@
-apiVersion: apiextensions.k8s.io/v1
```

Comment:
Could we just create a separate CRD YAML just for upgrade, but do not change this one that impacts the normal deployment?

Another question - could you remind me why we need two YAMLs for antrea-multicluster-leader-global.yml and antrea-multicluster-leader-namespaced.yml? I did not see where antrea-multicluster-leader-global.yml is used alone?

Reply:
Sure, I will keep the old one the same as before.
For the leader manifest, I think the reason is we support to run multiple leader controllers in one leader cluster for multi-tenant purpose. CRDs in antrea-multicluster-leader-global.yml can be installed or applied just one time by the admin in a leader cluster, but the namespaced deployment will be used for different customers.

Reply:
In my mind, even for leader we should generate one YAML, which is the simplest for the typical case. If there are really use cases of separate cluster and Namespace admins, customers can split the YAMLs themselves.